t/plugin/limit-conn.t

```diff
@@ -1207,9 +1207,9 @@ qr/limit key: consumer_jackroute&consumer\d+/
         content_by_lua_block {
             local plugin = require("apisix.plugins.limit-conn")
             local ok, err = plugin.check_schema({
-                conn = 1, 
-                default_conn_delay = 0.1, 
-                rejected_code = 503, 
+                conn = 1,
+                default_conn_delay = 0.1,
+                rejected_code = 503,
```

Comment:
why modify this test case？

Reply:
old test cases have `space` at the end of line. 

my editor helps me do the `slim`, this is fine.

![image](https://user-images.githubusercontent.com/6814606/101651756-5f0b6180-3a78-11eb-8ec3-18b9d8bd1704.png)